Sources/TokamakCore/Shapes/Path.swift

```diff
@@ -168,33 +193,222 @@ public enum RoundedCornerStyle: Hashable, Equatable {
   case continuous
 }
 
+public extension Path.Storage {
+  var elements: [Path.Element] {
+    switch self {
+    case .empty:
+      return []
+    case .rect(let rect):
+      return [
+        .move(to: rect.origin),
+        .line(to: CGPoint(x: rect.size.width, y: 0).offset(by: rect.origin)),
+        .line(to: CGPoint(x: rect.size.width, y: rect.size.height).offset(by: rect.origin)),
+        .line(to: CGPoint(x: 0, y: rect.size.height).offset(by: rect.origin)),
+        .closeSubpath
+      ]
+
+    case .ellipse(let rect):
+      // Scale down from a circle of max(width, height) in order to limit
+      // precision loss. Scaling up from a unit circle also looked alright,
+      // but scaling down is likely a bit better.
+      let size = max(rect.size.width, rect.size.height)
+      guard size > 0 else { return [] }
+      let transform: CGAffineTransform
+      if rect.size.width > rect.size.height {
+        transform = CGAffineTransform(scaleX: 1,
+                                      y: rect.size.height / rect.size.width)
+      } else if rect.size.height > rect.size.width {
+        transform = CGAffineTransform(scaleX: rect.size.width / rect.size.height,
+                                      y: 1)
+      } else {
+        transform = .identity
+      }
+      let elements =  [
```

Comment:
Nit: formatting is slightly off here. Could you run `swiftformat .` in the root directory with the latest SwiftFormat installed? You can also install a pre-commit hook [as described in `README.md`](https://github.com/TokamakUI/Tokamak#coding-style) to auto-format and lint all code before you commit.


```suggestion
      let elements = [
```

Reply:
I ran swiftformat manually for now. For some reason I get additional changed files when running in the pre-commit handler - even though it's using the same swiftformat configuration. Perhaps it's because my directory is not clean.